src/prefect/events/schemas/automations.py

```diff
@@ -429,4 +432,112 @@ class AutomationCore(PrefectBaseModel, extra="ignore"):
 
 
 class Automation(AutomationCore):
-    id: UUID = Field(..., description="The ID of this automation")
+    id: Optional[UUID] = Field(default=None, description="The ID of this automation")
```

Comment:
is this really something we expect anyone to initialize with?  Feels like a property that is read-only

Reply:
No worries -- yea probably not worth the added complexity, especially if we document `Automation.read` or `Automation.create` appropriately